services/buildbot/master/twisted_factories.py

```diff
@@ -523,7 +523,7 @@ class TwistedVirtualenvReactorsBuildFactory(TwistedBaseFactory):
     treeStableTimer = 5*60
 
     def __init__(self, source, RemovePYCs=RemovePYCs, python="python",
-                 trial="./bin/trial", virtualenv_module="virtualenv",
+                 trial="-m twisted.trial", virtualenv_module="virtualenv",
```

Comment:
Maybe not, but I don't have access to the builders, so I am only guessing
at how the buildbot logic works by looking at it.